zcl-builtin/matter/data-model/chip/matter-devices.xml

```diff
@@ -2042,4 +2042,60 @@ limitations under the License.
             </include>            
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-refrigerator</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Refrigerator</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0070</deviceId>
+        <class>Simple</class>
+        <scope>Endpoint</scope>
+        <endpointComposition>
+            <type> tree </type>
+            <endpoint mandatory ="true">
+                    <deviceType>0x0071</deviceType>
+            </endpoint>
+        </endpointComposition>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+            <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true"></include>
+            <include cluster="Refrigerator And Temperature Controlled Cabinet Mode" client="false" server="false" clientLocked="true" serverLocked="false">
+                <features>
+                    <feature code="" name="OnOff">
+                    <disallowConform/>
+                    </feature>
+                </features>
+            </include>
+            <include cluster="Refrigerator Alarm" client="false" server="false" clientLocked="true" serverLocked="false"></include>
+        </clusters>
+    </deviceType>
+    <deviceType>
```

Comment:
What is the purpose of this device type?

Reply:
It's a device type that consists of endpoint composition according to the spec

Reply:
fixed to include it and test it, thank you!